src/main/webapp/script.js

```diff
@@ -812,3 +815,43 @@ function zipcodelink() {
     var zipcode = document.getElementById("backToRepZipcode");
     zipcode.style.display = isRep.trim() == "true" ? "block" : "none"; 
 }
+
+function closeWindow(){
+    if(localStorage.getItem("rep").trim() == "true") {
+        var exit = confirm("Do you want to go offline?");  
+        if(exit == true) {
+            goOffline();
+        }
+    }
+}
+
+function goOnline(){
+        localStorage.setItem("online", true);
```

Comment:
how is this being persisted across user sessions? Localstorage only exists on a particular person's browser so how will that online/offline information stick around for users on other browsers?